provider/cloudflare.go

```diff
@@ -224,40 +218,27 @@ func (p *CloudFlareProvider) submitChanges(changes []*cloudFlareChange) error {
 }
 
 // changesByZone separates a multi-zone change into a single change per zone.
-func cloudflareChangesByZone(zones []cloudflare.Zone, changeSet []*cloudFlareChange) map[string][]*cloudFlareChange {
+func (p *CloudFlareProvider) cloudflareChangesByZone(zones []cloudflare.Zone, changeSet []*cloudFlareChange) map[string][]*cloudFlareChange {
```

Comment:
Method name could omit "cloudflare" in the name, because you already run it with cloudflare as receiver.

Reply:
Done